build/test-selenium.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pytest
+import time
+
+from notebook.notebookapp import list_running_servers
+from notebook.tests.selenium.utils import Notebook
+from selenium.common.exceptions import JavascriptException
+from selenium.webdriver import Firefox
+
+# workaround for missing os.uname attribute on some Windows systems
+if not hasattr(os, "uname"):
+    os.uname = lambda: ["Windows"]
+
+def setup_module():
```

Comment:
is this guaranteed to run in order?

Reply:
I think so. But I should just put this inside `setup_module()` anyway, it's cleaner that way.